guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -48,6 +48,10 @@ Boolean indicating whether the swipe gesture should work on whole screen. Swipin
 
 When set to `false` the back swipe gesture will be disabled. The default value is `true`.
 
+#### `gestureResponseDistance` (iOS only)
+
+Use it to restrict the distance from the edges of screen in which the gesture should be recognized. To be used alongside `fullScreenSwipeEnabled`.
```

Comment:
I think we should an example of the use here. From this description I couldn't tell what values does the prop take.